src/device-listing/Program.cs

```diff
@@ -56,6 +56,7 @@
     "reader",
     "satellite",
     "particulatematter",
+    "nox",
```

Comment:
Can this be just the same category as gas sensor?

Reply:
I'd indeed favor the existing category. No need to splinter up into even more categories unnecessarily. You could even consider consolidating "voc" into "gas" as well.